docs/guide/areas-and-widgets/custom-widgets.md

```diff
@@ -204,3 +204,42 @@ apos.util.widgetPlayers.accordion = {
 ```
 
 [Credit goes to Heydon Pickering](https://inclusive-components.design/collapsible-sections/) for the accessible collapser example.
+
+### Using widget data in players
+
+These widget players do not know anything about the data in the widget. If we want to use widget data in your player code, we will need to provide it.
+
+On the other hand, template files *do* have access to the data. One good way to use data in a widget player is to insert it as a data attribute value in the template. The player can then look for that data attribute.
+
+For example, we could change our collapser widget code to include a possible `color` field value:
```

Comment:
```suggestion
For example, we could change our collapser widget to include a `color` field value:
```

Reply:
@abea I'm not sure if this is worth pointing out, but the technical term for this is a disclosure widget. It has a complete spec, which accessibility tree, etc. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details